std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -171,22 +171,21 @@ public static Column fromRepeatedItem(
 
     Object converted = Polyglot_Utils.convertPolyglotValue(item);
 
-    Builder builder;
     if (converted == null) {
-      builder = new MixedBuilder(repeat);
+      Builder builder = new MixedBuilder(repeat);
+      builder.appendNulls(repeat);
+      return new Column(name, builder.seal());
     } else {
       StorageType storageType = StorageType.forBoxedItem(converted);
-      builder = Builder.getForType(storageType, repeat, problemAggregator);
-    }
-
-    Context context = Context.getCurrent();
+      Builder builder = Builder.getForType(storageType, repeat, problemAggregator);
+      Context context = Context.getCurrent();
+      for (int i = 0; i < repeat; i++) {
+        builder.appendNoGrow(converted);
+        context.safepoint();
+      }
 
-    for (int i = 0; i < repeat; i++) {
-      builder.appendNoGrow(converted);
-      context.safepoint();
+      return new Column(name, builder.seal());
```

Comment:
Hah, I actually did it because IMHO this is more readable - it clearly shows we have 2 branches that return some value, there is no non-local control flow. It's a bit more 'functional'. But I'm OK with changing this back if you prefer so.